### DISCUSSION TITLE
Can't seem to get some things to work

### DISCUSSION THREAD

**Discussion by ManuelSvQ** [winstxnhdw/lc-hax#117] [Q&A] [answered]
Phantom mode, weightless and stamina options do not work at all, I've tried to look at the code but I can't seem to be able to fix anything. **I need help!**

**Comment by ManuelSvQ** [comment]
Forgot to add, I cannot seem to get /stunclick working, what exactly do I have to do? Whenever I press M3 on an enemy it just moves them away, and when I click it does nothing, I'm just quite confused.

**Comment by winstxnhdw** [comment, marked as answer]
Try injecting after you've joined a game? Are you using other mods along with this?

**Reply by ManuelSvQ** [reply]
A bunch, I've always injected before clicking on Online, was I not meant to?

**Reply by winstxnhdw** [reply]
For whatever reason, certain stuff can break if you do that. We are still looking into it, but it's not a high priority at the moment since most people just don't do that.

**Reply by ManuelSvQ** [reply]
Injecting after joining fixed it, thank you.

**Comment by winstxnhdw** [comment]
> Phantom mode, weightless and stamina

@ManuelSvQ are these 3 the only mods that stop working? I am trying to tackle this issue.

**Reply by ManuelSvQ** [reply]
Some commands don't seem to work either, stun commands to be exact, I haven't tried it after fixing those mods by injecting in-game though
